random_fields/generate_field.py

```diff
@@ -28,26 +28,25 @@ class RandomFields():
         - None
 
     Attributes:
-        - max_conditioning_points (int):
-        - random_field_model_name (str):
+        - max_conditioning_points (int): maximum number of 
+        - random_field_model_name (str): 
         - random_field_model (int):
-        - n_dim (int):)
-        - seed (int):
-        - mean (float):
-        - variance (float):
-        - vertical_scale_fluctuation (list):
-        - anisotropy (list):
-        - angle (list):
-        - v_dim (int):
-        - random_field (:class:)
-        - z_kriged_field (array)
-        - GaussianProcess (:class:)
-        - conditioning_points (array):
-        - conditioning_values (array):
-        - kriging_mean (array):
-        - kriging_std = (array):
-        - conditioned_random_field (array):
-
+        - n_dim (int): number of physical dimesions (1,2 or 3) 
+        - seed (int): seed for the random number generator
+        - mean (float): unconditioned mean of the random field 
+        - variance (float): unconditioned variance of the random field
+        - vertical_scale_fluctuation (float): 
+        - anisotropy (list): ratio between horizontal scales and vertical scale of fluctuation, shape (`self.n_dims` - 1)
+        - angle (list): rotation angles of the principal directions of the scales of fluctuation relative to the vertical, shape (`self.n_dims` - 1)
+        - v_dim (int): dimesion number corresponding to the vertical scale of fluctuation
+        - random_field (:class:gstools.SRF) random field generator 
+        - z_kriged_field (array): standard-normal kriged random field
+        - gaussian_process (:class:sklearn.gaussian_process.GaussianProcessRegressor)
+        - conditioning_points (array): coordinates of the conditioning points, shape (:,`self.n_dims`)
+        - conditioning_values (array): values of the conditioning points, shape (:,)
```

Comment:
How to document arrays in the doc strings? I've seen "Sequence(float)" here and there... 

and what about shapes and defaults?